src/PackagePublisher/Program.cs

```diff
@@ -24,7 +24,9 @@ public static class Program
 
         // Do NOT log the key
         private static string _apiKey = null;
-        private static int _maxRetryCount = 10;
+        private static int _maxRetryCount = 5;
```

Comment:
Change `_packagePushTimeoutInSeconds` and `_semaphoreTimeoutInSeconds` from `private static int` to `private const int`.